front/app/containers/EventsShowPage/components/DesktopTopBar.tsx

```diff
@@ -39,10 +39,10 @@ interface Props {
 
 const TopBar = ({ project, event }: Props) => {
   const location = useLocation();
-  const user = useAuthUser();
+  const { data: user } = useAuthUser();
   const { formatMessage } = useIntl();
-  const isAdminUser = isAdmin(user.data);
-  const isModerator = isProjectModerator(user.data, project.id);
+  const isAdminUser = isAdmin(user);
```

Comment:
I don't see an issue with the refactoring but ideally, instead of having these checks in the components directly, we would use our `rules` mechanism